openapi.yaml

```
@@ -1436,6 +1431,9 @@ paths:
                   - rel: license
                     href: https://scihub.copernicus.eu/twiki/pub/SciHubWebPortal/TermsConditions/Sentinel_Data_Terms_and_Conditions.pdf
                     type: application/pdf
+                  - rel: http://www.opengis.net/def/rel/ogc/1.0/queryables
+                    href: https://example.openeo.org/api/collections/Sentinel-2A/queryables
```

```suggestion
                    href: https://openeo.example/api/collections/Sentinel-2A/queryables
```

(.example is a dedicated tld for examples, avoids risk of having a URL here that might start working in the future)

---

Fine, but I think it should be a separate issue that changes it for all examples. :-)

---

sure, feel free to merge this PR already

---

spin of: #489